wpilibjIntegrationTests/build.gradle

```diff
@@ -24,8 +24,11 @@ dependencies {
     compile 'com.googlecode.junit-toolbox:junit-toolbox:2.0'
     compile 'org.apache.ant:ant:1.9.4'
     compile 'org.apache.ant:ant-junit:1.9.4'
+    compile files(wpilibj.sourceSets.test.output.classesDir)
 }
 
+compileJava.dependsOn tasks.getByPath(':wpilibj:testClasses')
```

Comment:
what files are part of `testClasses`, and which ones are `compileJava`? I would think `testClasses` would be all the .java files in `wpilibjIntegrationTests` and `compileJava` was a task that compiled those? I feel like that dependency already exists for free in gradle